projects/observability/src/shared/components/utils/svg/svg-util.service.ts

```diff
@@ -150,13 +150,12 @@ export class SvgUtilService {
           this.truncateText(tspan.node()!, width);
         } else {
           // Move this word to next line
-
+          const y = textSelection.attr("y");
+          const dy = parseFloat(textSelection.attr("dy"));
           currentLine.pop();
           tspan.text(currentLine.join(' '));
           currentLine = [word];
-          tspan = appendTSpan()
-            .attr('dy', `${++lineNumber * 1.1}em`)
-            .text(word);
+          tspan = textSelection.append("tspan").attr("x", 0).attr("y", y).attr("dy", `${++lineNumber * 1.1 + dy}em`).text(word)
```

Comment:
how is `appendTSpan` different than `textSelection.append("tspan")` ?

Reply:
Oh sorry my mistake there 😅. No difference there, the change is after that, i.e., for the gap before the text in below line, and its line-height. Updating

Reply:
updated